prime-router/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml

```diff
@@ -0,0 +1,93 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: ../../../ORU_R01/ORU_R01-base
+
+elements:
+  # MSH 4-1
+  - name: sending-facility_namespace-id
+    condition: true
+    resource: 'Bundle'
+    value: [ '"CDC PRIME REPORTSTREAM"' ]
+    hl7Spec: [ MSH-4-1 ]
+
+  # MSH 4-2
+  - name: sending-facility-universal-id
+    resource: 'Bundle'
+    condition: true
+    value: [ '"10DRPTSTRM"' ]
+    hl7Spec: [ MSH-4-2 ]
+
+  - name: sending-facility-universal-id-type
+    resource: 'Bundle'
```

Comment:
This was a NY receiver transform requirement. I don't see this in AK receiver settings.